### DISCUSSION TITLE
Off grid working?

### DISCUSSION THREAD

**Discussion by phips** [StealthChesnut/HA-FoxESS-Modbus#30] [Q&A] [answered]
Hello folks,

Forgive the fact I'm asking this question here, but I'm struggling to find a decent place to ask. The internets doesn't come up with a lot of info for Fox ESS stuff that doesn't just end up on their own site. 

Has anyone here with a H1 inverter had it working with batteries during a power cut? (ie powering your home when the grid is down) The manuals suggest it's part of the product functionality (ESP mode) but mine doesn't and I'm trying to find a proof point to beat the installers with. 

Many thanks for any experience you can share. 

Loving the fact this project has enabled HomeAssistant integration none the less 😄

**Comment by Daibutt** [comment, marked as answer]
Unless you have an early model in which EPS is not enabled, the EPS port will work when AC is cut. 
It will not power your whole current house circuits, but you'll have to run a aeparate circuit (or reuse an existing one) and use that for  your key bits of kit that you want to stay powered in case of mains failure.
Also be aware of what types of load you put on this circuit, the manual gives some examples of loads that have starting loads 2-3x rated load, if these coincide and go over 5kW your EPS will drop out.
Some discussion here;
https://www.facebook.com/groups/foxessownersgroup/permalink/1578211785894331/


To power your whole house's existing circuits as is, you will need an external switch that completely isolates your house from the grid, so that people working on the mains network are not at risk from your power feeding back into the grid. I understand the Fox versions of this are available in DE but not in UK.

https://www.youtube.com/watch?v=8vqUIa1rzFo&list=PLectfpfs6rjZr07u8bNcM8QokKMT-tk_O&index=15

**Reply by phips** [reply]
That's really helpful, thank you @Daibutt. It's a brand new H1-5.0-E, and looking at the manual it appears it can be wired up either with or without an external EPS box. There's a port marked "backup" next to the AC output, which is probably the extra circuit required (it's currently not wired in 🙄 It's probably just not been done by the cowboy installers). I don't have Facebook so can't look at the comments there - if anyone mentions using an H1 without an external EPS box though I'd really appreciate a paste of the text in this thread if it's no trouble 😊 Thank you for your fast and helpful response!

**Comment by Daibutt** [comment]
@phips Yes definitely lots of people using EPS without the external box for critical loads on a separate circuit.
All the posts I've seen are on FB unfortunately & there is a video too. 
I did try copying text from one of the posts to paste here but it brings in all the html too, so no good.
Could you get someone you know with FB to open them up for you?
here's the link to another EPS post with the video;
https://www.facebook.com/groups/foxessownersgroup/permalink/1523124138069763/

**Comment by Daibutt** [comment]
@phips this is the adaptor that goes into backup port. Wire this to a sepaarte consumer unit or isolator switch for your critical load circuit.
![image](https://user-images.githubusercontent.com/69276073/180158863-93dd514b-01d4-40d6-bc52-aa0a41b3502d.png)

And here's a couple of stills from one of teh FB vids showing general setup.
![image](https://user-images.githubusercontent.com/69276073/180160719-ac4d677d-8b4a-4cb3-af14-e5aa8046fe72.png)

I've managed to copy text of one post without html, I'll post that next. It is quite lengthy.

**Comment by Daibutt** [comment]
@phips



 Cool video. I have done something similar. I will try and get you some pictures. From the inverter I am going into a garage consumer unit with RCD and MCB, then this runs to a twin socket downstairs in the utility cupboard.

    Reply
    21 w

William EcclesAuthor
Admin
Nice! I'm researching a change-over system which will automatically isolate the house from the grid during an outage and power the house off my two inverters so 10kW concurrent power support. I can run everything but charge my tesla.

    Reply
    21 w

Dez Futak
William Eccles - how did you get on with this change-over setup btw?

    Reply
    14 w

William EcclesAuthor
Admin
Dez Futak not yet had the auto change over installled. Looks like an £600 investment

    Reply
    14 w

Dez Futak
William Eccles - separate piece of kit to Fox-ESS I'm guessing?

    Reply
    14 w

Lou JenningsGroup expert
Dez Futak I've not found the EPS-BOX proper part available in the UK. Chatting with an electrician, they are used to doing something like this with petrol generators using a contactor, to break the grid circuit and then make the house consumer unit join to the EPS feed.

    Reply
    14 w

Dez Futak
Lou Jennings - cool. £600 seems like a v good price to have it done by a sparky, I must say

    Reply
    14 w

William EcclesAuthor
Admin
Dez Futak sadly yes

    Reply
    14 w

William EcclesAuthor
Admin
Lou Jennings that is exactly what you need for this

        Reply
        14 w


William EcclesAuthor
Admin
Does anyone know what connector type this is or a link to buy another, my parents are missing theirs and I can't seem to find it.

    Reply
    21 w

Lou JenningsGroup expert
William Eccles would you have any joy asking your installer for a spare or 2?

    Reply
    21 w

William EcclesAuthor
Admin
Lou Jennings doubt it, they rubbish. 🙂

            Reply
            21 w

Lou JenningsGroup expert
William Eccles great minds...
0:00 / 1:19

    Reply
    21 w

William EcclesAuthor
Admin
Lou Jennings that is a really neat idea! I'm going to copy you and do the same

    Reply
    21 w

William EcclesAuthor
Admin
Lou Jennings any idea of the connector type, name or make. I'm trying to locate an extra plug for my parents and i'm not finding the right one online. Mine has Devalan-4 printed on the side and the bag my came in had a barcode 10-208-00009-00 but drawing a blank 🙁

    Reply
    21 w

Lou JenningsGroup expert
William Eccles let me see what I can find out. I know some people...

    Reply
    21 w

William EcclesAuthor
Admin
Lou Jennings thank you

    Reply
    21 w

Dave StuckeyGroup expert
William Eccles ditto same barcode, scanning the QR didn't get anything either.

    Reply
    21 w

William EcclesAuthor
Admin
Dave Stuckey it's not a foxess special i'm sure it's a standard plug type. checking their datasheet doesn't give me any more clues and foxess don't have a spares area on their site 🙁

    Reply
    21 w

Lou JenningsGroup expert
William Eccles Did you see my direct message to you about this? 🙂

        Reply
        19 w

William EcclesAuthor
Admin
Lou Jennings just thinking about my backup socket design, I think I might also add a 13amp isolator switch that has a red led warning lamp integrated to add an easy point of isolation for the socket without needing to go up into the loft. It would also act as a warning device should someone in the future attempt to work on the socket. Thoughts? Overkill?

    Reply
    21 w

Lou JenningsGroup expert
You need something between the inverter and socket to isolate and overload protect. I went for the consumer unit in the attic. I nearly put the consumer unit near to the socket, but wanted it out of site and hopping into the attic is not a big issue. You could put one of the fused spurs in to protect the inverter from too much draw by way of the fuse, and to isolate by way of the switch. I think I'd recommend a RCD protected socket too, if you're doing this as an alternative to the consumer unit.
No photo description available.

    Reply
    21 wEdited

Lou JenningsGroup expert
No photo description available.

        Reply
        21 w


Dave StuckeyGroup expert
Wow this is good. William Eccles do you have to toggle any of these to let the inverter know to power up the EPS outlet?
May be an image of text that says "EpsOutput PowerLimit 100 ByPassRelay External Grounding EpsRatedVoltage 230 EpsOverloadAutoRecovery *IdleState *UpsOutput"

    Reply
    21 wEdited

William EcclesAuthor
Admin
Dave Stuckey yes, EpsOutput needs to be enabled which can be turned on via the inverter directly under off-grid menu or via that agent screen.

    Reply
    21 w

Dave StuckeyGroup expert
William Eccles Awesome, thanks.👍

    Reply
    21 w

Lou JenningsGroup expert
William Eccles I think if I'm not mistaken, bypass relay is internal for this direct connection. We'd only use external if we had the box of tricks to do what you have in mind for your future project of entire house supply.

    Reply
    21 w

William EcclesAuthor
Admin
Lou Jennings correct, you need a "change over" switch to cut the supply to do whole home backup. I'm really only interested in a couple sockets like what you've done to power a few devices.

    Reply
    21 w

William EcclesAuthor
Admin
Dave Stuckey Bypass relay set to internal and UPS set to enabled appears to allow the inverter to act like a proper UPS so the device like a server or internet router says on even if the grid power if going on and off! neat! neat! neat!

    Reply
    21 w

Dave StuckeyGroup expert
William Eccles Sorry Will I didn't see this post before I asked my question.
Off to Screwfix in am to pick up consumer unit (as per Lou Jennings) and cabling/outlets. Let the carpet pulling up begin! Happy days😕. Thanks both, this has been a bit of a revelation.

    Reply
    21 w

Lou JenningsGroup expert
Dave Stuckey 1 x Chint NX3 5-Module 2-Way Populated Garage Consumer Unit (9864V)

    Reply
    21 w

Dave StuckeyGroup expert
Lou Jennings 9864V x 1 Chint NX3 5-Module 2-Way Populated Garage Consumer Unit £33.60 on order ... your video was great.. let me find the exact same one. Thanks!

        Reply
        21 w


Dave StuckeyGroup expert
William EcclesLou Jennings Possibly a stupid question I guess but, is the EPS outlet only live when AC power is cut to inverter, or can I use it like a UPS. i.e. powered all the time and just switches to battery when AC is out?

    Reply
    21 w

William EcclesAuthor
Admin
Dave Stuckey If you have EPSOutlet On, UPS On and Relay set to internal then it will act like a proper UPS!

    Reply
    21 w

Dave StuckeyGroup expert
😲 omfg... well that is awesomely cool. When I asked installer rep about backup power he said it was very difficult & not worth doing (which I guess is partly true if you want to run the whole house seamlessly). But I never realised you could have a separate supply which was powered/UPS backup.
Just been walking through house now with tape measure working out how I can get cable down from loft to router/NAS/HA box/alarm box/Hive etc without lifting all my carpets & floor boards lol.

    Reply
    21 w

Lou JenningsGroup expert
Dave Stuckey I know right. I was told the same this, but that's because they only know about the box of tricks that does the whole house.

    Reply
    21 w

Lee Phillips
William Eccles I'm finally getting my H1 3.7 installed on Monday. The batteries are delayed (by a couple of weeks?) but eventually there will be 10.4 kWh.
Question about the EPS/UPS with a view to having the installer fit a critical loads consumer unit...
If the mains is live, will the EPS pass through more than 21.7A (5kVA)? This would allow me to connect the kitchen ring main, which has the washer/tumbler/dishwasher on it. If the mains is off then we can limit what gets used to <5kVA
My thinking is that I can get whole circuits moved over to it, rather than have to select a limited number of loads/run new circuits or just wire the EPS to a 2-gang socket.
Also, if the solar input is high (>3.7kW), will the inverter output 3.7VA to the grid and 5kVA to the EPS simultaneously?
I'll experiment once the batteries are installed but just wondered whether this has been explored.
Cheers

        Reply
        14 w

Phil Stinger
Dave Stuckey Where do you get these options please?

    Reply
    21 w

Lou JenningsGroup expert
Phil Stinger most of us have to change these on the inverter panel itself user the settings menu as we have End User accounts. It seems Dave has elevated access.

    Reply
    21 w

Dave StuckeyGroup expert
Phil Stinger Lou Jennings I am on Agile incoming/outgoing so need to change my charge times daily at least, depending when cheapest slots are. When I had my install ~ 9 months ago, the app & WebUI couldn't do this with User access, so my installer had to request Fox to set me up with Agent access.

    Reply
    21 w

Filip Jurik
Dave Stuckey I asked about Agent access and been told big fat NO, but then have been given installer access. I think they might be more less same.

    Reply
    15 w

William EcclesAuthor
Admin
Who gave you installer profile? Foxess or your agent?

    Reply
    15 w

Filip Jurik
William Eccles FoxESS technician

    Reply
    15 w

Filip Jurik
William Eccles basically he unbind my inverter from my account and I had to register new account as installer.

    Reply
    15 w

William EcclesAuthor
Admin
Filip Jurik yeh that is the process a few of us have been through

    Reply
    15 w

Filip Jurik
William Eccles I need your help, can I PM you?

    Reply
    15 w

William EcclesAuthor
Admin
Filip Jurik sure thing, shoot me a message on messager and I'll accept your message

        Reply
        15 w



William EcclesAuthor
Admin
Eccles the master investigator has figured out the connector type. I found my orginal packaging that my connectors came in and it had Vaconn power on the bag..... and bingo! here is the connector kit you need for EPS should you be missing the plug. http://www.vaconn.cn/enShowProducts.asp?id=65
Dianwei Tech
VACONN.CN
Dianwei Tech
Dianwei Tech

    Reply
    21 w

Lou JenningsGroup expert
William Eccles that's great. We ought to stick all these in a master word document. Next job, find an seller of these connectors....

    Reply
    21 w

William EcclesAuthor
Admin
Lou Jennings i've found some places that sell them but only ship to germany. So if anyone has any family in germany that could relay this into the UK let me know.

    Reply
    21 w

Lou JenningsGroup expert
William Eccles sent you a private message...

    Reply
    19 w

William EcclesAuthor
Admin
Lou Jennings thanks

            Reply
            19 w


Chris Jeffrey
Did you guys ever manage to find a way of getting one the EPS connectors? I am assuming, as mine wasn’t needed at the moment of install the installers through it out with the boxes. I can’t seem to find any for sale. (I might not be googling the correct terms though!)

    Reply
    2 w

Peter Van Engelandt
Chris Jeffrey it's included with the inverter, but mostly installer take unused things with them, luckily i observed it and kept it for future install also the plastic tool to make the connection loose again

    Reply
    2 w

Chris Jeffrey
Hi Peter Van Engelandt, yes I figured that the installer must have taken them with them. I was wondering if anyone on this group had found anywhere to purchase them from for people who where not sharp enough to get the installer to leave them.

**Reply by phips** [reply]
That's incredibly helpful @Daibutt, thank you! (oder, danke 🇩🇪 wenn du bist deutsch)
